lua/lvim/config/defaults.lua

```diff
@@ -24,6 +24,10 @@ return {
     -- use config.lua for this not put here
   },
 
+  lazy = {
+    opts = {},
```

Comment:
put [the defaults](https://github.com/LunarVim/LunarVim/blob/2d373036493b3a61ef24d98efdeecbe8e74467be/lua/lvim/plugin-loader.lua#L96-L121) here, this way completion will work